scalingo/data_scalingo_database_firewall_managed_range.go

```diff
@@ -0,0 +1,77 @@
+package scalingo
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/Scalingo/go-scalingo/v8"
+)
+
+func dataSourceScDatabaseFirewallManagedRange() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceScDatabaseFirewallManagedRangeRead,
+		Description: "Database firewall managed range retrieved from the Database API",
+
+		Schema: map[string]*schema.Schema{
+			"database_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "ID of the Database NG",
```

Comment:
issue: I don't think we should use acronyms such as "NG" in public documentation. Moreover, is "next gen" the public name of this feature? I mean, naming a product "next" something is not ideal. It can be outdated quickly

```suggestion
				Description: "ID of the Database next gen",
```